lib/src/types/video_parameters.dart

```diff
@@ -26,7 +26,7 @@ class VideoParameters implements Comparable<VideoParameters> {
   const VideoParameters({
     this.description,
     required this.dimensions,
-    required this.encoding,
+    this.encoding = VideoEncoding.defaults,
```

Comment:
encoding is dimension-specific.. I don't think we should set it to the same across the board. better to make it optional.. and use presets?